### DISCUSSION TITLE
Visual Studio 2022 preview is not supported yet?

### DISCUSSION THREAD

**Discussion by mingtong** [microsoft/WindowsAppSDK#1021] [Q&A] [answered]
I was failed to install the  Project Reunion on my Windows 11 with VS 2022 preview installed from the link below
https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftProjectReunion

**Comment by mrlacey** [comment, marked as answer]
The extension currently only supports VS2019.
Support for VS2022 (Dev17) [has been done](https://github.com/microsoft/WindowsAppSDK/pull/936) but not yet released

**Reply by dyoalie** [reply]
So how soon will it be released? I just wanna try in VS 2022😋
